Tools/TestWebKitAPI/Tests/WebKitCocoa/ServiceWorkerBasic.mm

```diff
@@ -1933,6 +1938,107 @@ void testSuspendServiceWorkerProcessBasedOnClientProcesses(UseSeparateServiceWor
     done = false;
 }
 
+@interface CryptoKeyWebsiteDataStoreDelegate: NSObject <_WKWebsiteDataStoreDelegate> {
+@private
+    NSInteger keyFetchCount;
+}
+- (NSData *)webCryptoMasterKey;
+- (NSUInteger)fetchCount;
+@end
+
+@implementation CryptoKeyWebsiteDataStoreDelegate
+- (NSData *)webCryptoMasterKey {
+    // Not so random key
+    constexpr size_t keyLength = 16;
+    uint8_t keyBytes[keyLength] = { 1 };
+    NSData * key = [NSData dataWithBytes:keyBytes length:(keyLength)];
+    keyFetchCount += 1;
+    return key;
+}
+
+- (NSUInteger)fetchCount {
+    return keyFetchCount;
+}
+
+@end
+
+@interface SWCryptoKeyMessageHandler : NSObject <WKScriptMessageHandler>
+@end
+
+@implementation SWCryptoKeyMessageHandler
+- (void)userContentController:(WKUserContentController *)userContentController didReceiveScriptMessage:(WKScriptMessage *)message
+{
+    WTFLogAlways("%s\n", [[message body] UTF8String]);
```

Comment:
For debugging, I mean actively debugging an existing issue (e.g. the test are already flaky on bots and cannot be reproduced locally), and that should be a temporary measure (i.e. we will remove the log as soon as we figure out what's wrong, otherwise the bot's result will be noisy, and bot can be slowed down). This is not the case.

Reply:
> bot's result will be noisy, and bot can be slowed down
Thanks for explaining. I get that. I will remove it.